test/Microsoft.OpenAPI.OData.Reader.Tests/OpenApiExtensions/OpenApiDeprecationExtensionTests.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.IO;
+using Microsoft.OpenApi.Interfaces;
+using Microsoft.OpenApi.Writers;
+using Xunit;
+
+namespace Microsoft.OpenApi.OData.OpenApiExtensions.Tests;
+
+public class OpenApiDeprecationExtensionTests
+{
+    [Fact]
+    public void ExtensionNameMatchesExpected()
+    {
+        // Arrange
+        OpenApiDeprecationExtension extension = new();
+
+        // Act
+        string name = extension.Name;
+        string expectedName = "x-ms-deprecation";
+        Assert.Equal(expectedName, name);
```

Comment:
why duplicate the Assert.Equal(...)

Reply:
I got too enthusiastic with GitHub copilot, solved.